src/minectl.ts

```diff
@@ -0,0 +1,243 @@
+          args: { name: "plugin" },
+        },
+      ],
+    },
+    {
+      name: "rcon",
+      description: "RCON client to your Minecraft server",
+      options: [
+        {
+          name: ["--filename", "-f"],
+          description: "Contains the configuration for minectl",
+          args: { name: "filename" },
+        },
+        {
+          name: "--id",
+          description: "Contains the server id",
+          args: { name: "id" },
+        },
+      ],
+    },
+    {
+      name: "update",
+      description: "Update an Minecraft Server",
+      options: [
+        {
+          name: ["--filename", "-f"],
+          description: "Contains the configuration for minectl",
+          args: { name: "filename" },
+        },
+        {
+          name: "--id",
+          description: "Contains the server id",
+          args: { name: "id" },
+        },
+      ],
+    },
+    { name: "version", description: "Display the clients version information" },
+    {
+      name: "wizard",
+      description:
+        "Calls the minectl wizard to create interactively a minectl 🗺 config",
+      options: [
+        {
+          name: ["--output", "-o"],
+          description:
+            "Output folder for the configuration file for minectl 🗺 (default: /Users/dirien/.minectl)",
+          args: { name: "output" },
+        },
+      ],
+    },
+    {
+      name: "help",
+      description: "Help about any command",
+      subcommands: [
+        {
+          name: "completion",
+          description:
+            "Generate the autocompletion script for the specified shell",
+          subcommands: [
+            {
+              name: "bash",
+              description: "Generate the autocompletion script for bash",
+            },
+            {
+              name: "fish",
+              description: "Generate the autocompletion script for fish",
+            },
+            {
+              name: "powershell",
+              description: "Generate the autocompletion script for powershell",
+            },
+            {
+              name: "zsh",
+              description: "Generate the autocompletion script for zsh",
+            },
+          ],
+        },
+        { name: "create", description: "Create an Minecraft Server" },
+        { name: "delete", description: "Delete an Minecraft Server" },
+        { name: "list", description: "List all Minecraft Server" },
+        {
+          name: "plugins",
+          description: "Manage your plugins for a specific server",
+        },
+        { name: "rcon", description: "RCON client to your Minecraft server" },
+        { name: "update", description: "Update an Minecraft Server" },
+        {
+          name: "version",
+          description: "Display the clients version information",
+        },
+        {
+          name: "wizard",
+          description:
+            "Calls the minectl wizard to create interactively a minectl 🗺 config",
+        },
+      ],
```

Comment:
This template is a nice sugar that does the same thing requiring not maintenance
```suggestion
      args: {
        template: "help",
      },
```